src/curated-corpus/components/SidebarDatePicker/SidebarDatePicker.tsx

```diff
@@ -30,22 +28,34 @@ export const SidebarDatePicker: React.FC<SidebarDatePickerProps> = (
 ): JSX.Element => {
   const { handleDateChange, selectedDate } = props;
 
+  // state variable to track date picker open state.
+  // This is being controlled by the onClick on the TextField component within the DatePicker component below
+  const [open, setOpen] = useState(false);
```

Comment:
@nina-py thoughts on this?

Reply:
Nice thing to have, but do we really need it? Two things I can see that create extra maintenance workload here:

1. If we've added this functionality in the sidebar date picker, we should have it everywhere the date pickers are used. And is it worth going and adding extra state variables elsewhere, e.g. on the scheduling form, the scheduling page, anywhere else?
2. If we've added this functionality in the sidebar date picker, we need unit tests for it, too. There are none coming with this PR. And we'd need test coverage for the other places the date pickers are being modified, too.

And so it becomes a question: is it really worth having? I'd stick to default MUI functionality as much as possible. Especially when working with an officially "unstable" component whose API may change in the near future. Getting it to work if something breaks after a library update will be a lot easier if we don't add additional customisations to it.